use-cases/xtclim/preprocessing/preprocess_2d_seasons.py

```diff
@@ -112,8 +112,8 @@ def execute(self):
         scenarios = [self.scenario]
         # Load preprocessed "daily temperature images" and time series
 
-        train_images = np.load("input/preprocessed_2d_train_data_allssp.npy")
-        test_images = np.load("input/preprocessed_2d_test_data_allssp.npy")
+        train_images = np.load("input/preprocessed_2d_train_data_allssp.npy", allow_pickle=True)
+        test_images = np.load("input/preprocessed_2d_test_data_allssp.npy", allow_pickle=True)
         train_time = pd.read_csv("input/dates_train_data.csv")
         test_time = pd.read_csv("input/dates_test_data.csv")
```

Comment:
Best practice here would be to use pathlib for handling the paths. It could also be better to store them in variables so they are easier to change in the future, but not super important. Calling them e.g. `train_image_path` would be nice.